assets/src/edit-story/components/dropTargets/provider.js

```diff
@@ -30,6 +30,9 @@ import Context from './context';
 const DROP_SOURCE_ALLOWED_TYPES = ['image', 'video'];
 const DROP_TARGET_ALLOWED_TYPES = ['image', 'video', 'shape'];
 
+const isDropSource = (type) => DROP_SOURCE_ALLOWED_TYPES.includes(type);
```

Comment:
/cc @swissspidy and @cvolzke4  I'd really like a linter for this. IMHO this is a super-low-hanging-fruit that we run into all the time.

Reply:
Let's discuss this in #1776. Still don't now any existing linter rule for this, but worst case we could try to build one ourselves.